src/main/java/de/kekru/struktogrammeditor/control/Controlling.java

```diff
@@ -140,26 +138,27 @@ public Struktogramm neuesStruktogramm(){
 
 	public void speichern(boolean neuenSpeicherpfadAuswaehlenLassen){
 		Struktogramm str = gibAktuellesStruktogramm();
-
 		if(str != null){
 			GlobalSettings.setzeSpeicherpfad(str.speichern(neuenSpeicherpfadAuswaehlenLassen,GlobalSettings.getZuletztGenutzterSpeicherpfad()));//Struktogramm wird gespeichert (zuletztGenutzterSpeicherpfad wird dabei übergeben, damit der JFileChooser, sofern er genutzt wird, dort startet) und der neue Speicherpfad wird gesichert
 			GlobalSettings.saveSettings();
 			titelleisteAktualisieren();
 		}
 	}
 
+	/**
+	 * Loads a Struktogramm
+	 */
 	public void laden(){
+		File pfad = Struktogramm.oeffnenDialog(GlobalSettings.getZuletztGenutzterSpeicherpfad(), gui);//Parameter ist der Startordner für den OpenDialog
 
-		String pfad = Struktogramm.oeffnenDialog(GlobalSettings.getZuletztGenutzterSpeicherpfad(), gui);//Parameter ist der Startordner für den OpenDialog
-
-		if(!pfad.equals("")){
+		if(pfad != null){

```

Comment:
Do we need to check if File exists, as in line 57?

Reply:
I'd recommend it since the user may just have typed a fancy file name in the dialog...